scout/quickstart.md

```diff
@@ -0,0 +1,218 @@
+---
+title: Docker Scout quickstart
+keywords: scout, supply chain, vulnerabilities, packages, cves, scan, analysis, analyze
+description: >
+  Learn how to get started with Docker Scout to analyze images and fix vulnerabilities
+redirect_from:
+  - /atomist/get-started/
+---
+
+{% include scout-early-access.md %}
+
+Docker Scout analyzes image contents and generates a detailed report of packages
+and vulnerabilities that it detects. It can provide you with
+suggestions for how to remediate issues discovered by image analysis.
+
+
+This guide takes a vulnerable container image and shows you how to use Docker
+Scout to identify and fix the vulnerabilities, compare image versions over time,
+and share the results with your team.
+
+## Setup
+[This example project](https://github.com/docker/scout-demo-service) contains 
+a vulnerable Node.js application that you can use to follow along.
+
+1. Clone its repository:
+
+    ```shell
+    git clone https://github.com/docker/scout-demo-service.git
+    ```
+
+2. Move into the directory:
+
+    ```shell
+    cd scout-demo-service
+    ```
+
+3. Build the image, naming it to match the organization you will push it to, 
+and tag it as "v1":
+
+    ```shell
+    docker build -t <org-name>/scout-demo:v1 .
+    ```
+
+4. Create and push the repository on Docker Hub:
+
+    ```shell
+    docker push <org-name>/scout-demo:v1
+    ```
+
+    > **Important**
+    >
+    > Make sure you login to the Docker CLI or Docker Desktop before pushing.
```

Comment:
```suggestion
    > Make sure you log in to the Docker CLI or Docker Desktop before pushing.
```

Reply:
```suggestion
    > Make sure you sign in to the Docker CLI or Docker Desktop before pushing.
```

Reply:
@usha-mandya Happy to change this, but I can see the docs using "login" in a LOT of places, so should we also do a bulk update and some Vale rules etc?

Reply:
@ChrisChinchilla It should be two words when used as a verb. Also, we recommend using "sign in" instead of "log in" as per our style guide. Didn't pick this up in my first pass. My bad.  Maybe we should add this rule to Vale?